packages/plugin-interaction-breadcrumbs/interaction-breadcrumbs.js

```diff
@@ -53,8 +82,36 @@ function getNodeSelector (el, win) {
   return parts.join('')
 }
 
-function truncate (value, length) {
-  const ommision = '(...)'
-  if (value && value.length <= length) return value
-  return value.slice(0, length - ommision.length) + ommision
+function isWhitespace (character) {
+  switch (character) {
+    case ' ': // space
+    case '\n': // newline
+    case '\r': // carriage return
+    case '\t': // tab
+    case '\f': // linefeed
+    case '\v': // vertical tab
+    case '\u00a0': // no-break space
+    case '\u1680': // ogham space mark
+    case '\u2000': // en quad
+    case '\u2001': // em quad
+    case '\u2002': // en space
+    case '\u2003': // em space
+    case '\u2004': // three-per-em space
+    case '\u2005': // four-per-em space
+    case '\u2006': // six-per-em space
+    case '\u2007': // figure space
+    case '\u2008': // punctuation space
+    case '\u2009': // thin space
+    case '\u200a': // hair space
+    case '\u2028': // line separator
+    case '\u2029': // paragraph separator
+    case '\u202f': // narrow no-break space
+    case '\u205f': // medium mathematical space
+    case '\u3000': // ideographic space
+    case '\ufeff': // zero width no-break space
```

Comment:
This matches the previous behaviour[^1], but is maybe excessive — we could save a lot of bytes by not counting more esoteric whitespace here. I think the only common values would be ` `, `\t`, `\n` & `\r` and especially only those as repeating characters — i.e. I'd expect multiple ` ` or `\t` for indentation and `\n` & `\r` for new lines, but I wouldn't expect many repeated hair spaces

[^1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#white_space